Networking/Sources/Networking/TargetType.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TargetType.swift
+//  
+//
+//  Created by marwa on 28/06/2022.
+//
+
+import Foundation
+import Alamofire
+
+/// Used to map alamofire http methods.
+///
+/// - Note: so if there any change in alamofire, we need to update it here.
+public enum HTTPMethodd {
+    case get
+    case post
+    case put
+    case delete
+    var httpMethod: HTTPMethod {
+        switch self{
+        case .get:
+            return .get
+        case .post:
+            return .post
+        case .put:
+            return .put
+        case .delete:
+            return .delete
+        }
+    }
+}
```

Comment:
It's a kind of duplication.. Why not using the original `Alamofire.HTTPMethod` directly?